splunk-otel-android/src/main/java/com/splunk/rum/AppStartupTimer.java

```diff
@@ -78,6 +78,7 @@ void setCompletionCallback(Runnable completionCallback) {
     }
 
     void end() {
+        Span overallAppStartSpan = this.overallAppStartSpan;
```

Comment:
This is an interesting fix to an easily overlooked problem! Did one of the tools catch this?

Reply:
Yep! NullAway actually -- because the second time you reference the field, it might as well have a different value.